confd/templates/production.tmpl

```diff
@@ -255,13 +256,38 @@
       "geoFilterEdit": "Edit Geo filter",
       "geoFilterShow": "Show/hide Geo filter",
       "timeFilter": "Time filter",
+      "dataInputFilter": "Data Input Filter",
```

Comment:
Maybe we should rename it to "Edit data input filters" for consistency.

Reply:
Look at the other filters... none of them has "Edit"...
I don't think that's a good idea...